src/xcode/ENA/ENA/Source/Models/Settings/SettingsViewModel.swift

```diff
@@ -17,48 +17,53 @@
 
 import Foundation
 
-class SettingsViewModel {
+final class SettingsViewModel {
+	// MARK: Properties
 	var tracing: Main
 	var notifications: Main
 	var reset: String
 
-	init(tracing: Main, notifications: Main, reset: String) {
+	// MARK: Creating a Settings View Model
+	init(
+		tracing: Main = .tracing,
+		notifications: Main = .notifications,
+		reset: String = AppStrings.Settings.resetLabel
+	) {
 		self.tracing = tracing
 		self.notifications = notifications
 		self.reset = reset
 	}
-
-	static let model = SettingsViewModel(
```

Comment:
This was a hidden singleton in our app. It was discovered by writing a couple of unit tests for this class. I was able to simply replace `SettingsViewModel.model` by `SettingsViewModel()` because `SettingsViewModel.model` was only used once in our app. So we did not really use it as a singleton.